### ISSUE TITLE
Let user set the sense of rotation (Struct CoordSystem) for ArcBall

### ISSUE THREAD

**Issue by Nevsden** [sebcrozet/kiss3d#238]
**TL;DR**
Consider making struct `CoordSystem` public and available with `pub use self::arc_ball::CoordSystem`.
Also extend the constructor `ArcBall::new_with_frustrum` to set `coord_system: CoordSystem` so that the user can decide around which of both Y/Z axis to roll around.

# Issue
When building a 3D geometry viewer and using ArcBall as the camera, I was kind of annoyed to find out that camera rotation around Y-axis and Z-axis are interchanged. By default ArcBall rolles around the Y-axis. Also by default, rotating around the Z-axis is not implemented. 

My hotfix was to edit these files:

File `/src/camera/mod.rs`
```rust
//! Camera trait with some common implementations.

pub use self::arc_ball::{ArcBall, CoordSystem};
...
```

File `src/camera/arc_ball.rs`
```rust
pub fn new_with_frustrum(
        fov: f32,
        znear: f32,
        zfar: f32,
        eye: Point3<f32>,
        at: Point3<f32>,
        coord_system: CoordSystem,
    ) -> ArcBall {
          ...
          coord_system: coord_system,
          ...
}
...
pub enum CoordSystem {
...
}
```

**Comment by alvinhochun** [comment]
I believe calling `ArcBall::set_up_axis_dir` with `Vector3::z_axis()` should already work for you, assuming what you are using is a right-handed Z-up coordinate system. No need to expose the `CoordSystem` type (it's supposed to be an implementation detail).

**Comment by Nevsden** [comment]
Ah, thank you so much! Did not know that.
